src/constants.ts

```diff
@@ -1,2 +1,3 @@
 // eslint-disable-next-line import/prefer-default-export
 export const CURRENT_SCHEMA_VERSION = 2;
+export const URL_USER_FEEDBACK_FORM = 'https://forms.gle/YNozsPZxF4kxSuud8';
```

Comment:
I opened this form and it asks if I want to provide a bug report or general feedback. Since the bug reports are covered by the github issues system now, I think we should change the google form to just be for general feedback.

Reply:
Also, is this form in our shared drive or somewhere else? I wanted to make a couple of spelling corrections but couldn't find it

Reply:
Good point. Will mention this to team phoenix. The survey link is in the team phoenix chat

Reply:
Also, will remove the bug-report part of the survey once a PR is approved for this feature